### DISCUSSION TITLE
Agent Manager for UTMStack swarm networks 10.0.0.0/24

### DISCUSSION THREAD

**Discussion by TYrorare** [utmstack/UTMStack#590] [Q&A] [answered]
After installing UTMStack Was created overlay network with name ingress. That network has 10.0.0.0/24 CIDR.
I have local network the same CIDR . And UTMStack server has ip 10.0.0.27
In that case I can't connect any agents to the server. 
I try to recreate ingress network with another CIDR, but unsuccessful
Can somebody explaine me how to change this swarm overlay network?

**Comment by lacak-sk** [comment, marked as answer]
Can you test steps described in https://github.com/orgs/utmstack/discussions/599#discussioncomment-9260240
